docs/source/ja/_toctree.yml

```
@@ -11,4 +11,7 @@
 - sections:
   - sections:
     - local: multilingual
-      title: 推論のための多言語モデル
\ No newline at end of file
+      title: 推論のための多言語モデル
+- sections:
+  - local: autoclass_tutorial
+    title: AutoClassを使用してポータブルなコードを書
```

```suggestion
    title: AutoClassを使用してポータブルなコードを書く
```
Maybe you meant ポータブルなコードを書く instead of ポータブルなコードを書 ?